src/client/Microsoft.Identity.Client/Platforms/iOS/IntuneEnrollmentIdHelper.cs

```diff
@@ -42,6 +42,16 @@ internal static string GetEnrollmentId(ICoreLogger logger)
             return string.Empty;
         }
 
+        internal static string GetRawEnrollmentId()
+        {
+#if iOS
+            var keychainData = NSUserDefaults.StandardUserDefaults.StringForKey(EnrollmentIdKey);
```

Comment:
Can this be unit tested?

Reply:
It is iOS specific. So it cannot be unit tested.